packages/babel-helpers/src/helpers.js

```diff
@@ -432,13 +432,29 @@ helpers.objectDestructuringEmpty = template(`
 `);
 
 helpers.objectWithoutProperties = template(`
-  (function (obj, keys) {
+  (function (source, excluded) {
+    if (source === null) return {};
```

Comment:
Spec says we do [link](https://tc39.github.io/proposal-object-rest-spread/#AbstractOperations-CopyDataProperties) (point 3). But we should be returning the target (so instead of an empty object, it'd be null). Fixed in latest commit